docs/sources/shared/alerts/alerting_provisioning.md

```diff
@@ -1742,8 +1742,8 @@ Status: Accepted
 
 | Name        | Type                      | Go type | Required | Default | Description | Example |
 | ----------- | ------------------------- | ------- | :------: | ------- | ----------- | ------- |
-| EndMinute   | int64 (formatted integer) | `int64` |          |         |             |         |
-| StartMinute | int64 (formatted integer) | `int64` |          |         |             |         |
+| end_time    | int64 (formatted integer) | `int64` |          |         |             |  `"end_time": "24:00"` |
```

Comment:
@tonypowa 

According to this [line of code](https://github.com/grafana/grafana/blob/c0058f9c7e390d8a196f5b375382334287633ea9/pkg/services/ngalert/api/tooling/definitions/provisioning_mute_timings.go#L155), it seems `Type` and `Go type` should be `string`

Reply:
good catch @ppcano  thank you , I will update those too